sql/rowexec/agg.go

```diff
@@ -117,6 +117,8 @@ type groupByGroupingIter struct {
 	pos           int
 	child         sql.RowIter
 	dispose       sql.DisposeFunc
+	keyRow        sql.Row
+	keySch        sql.Schema
```

Comment:
(minor) Would be nice to add a comment for these explaining that they are just buffers to prevent having to reallocate memory.